fastlane/Fastfile

```diff
@@ -217,12 +226,13 @@ REPOSITORY_NAME="WordPress-Android"
   #####################################################################################
   desc "Updates a release branch for a new beta release"
   lane :new_beta_release do | options |
+    app = options[:app]
     android_betabuild_prechecks(options)
     send_strings_for_translation()
-    download_translations(options)
-    android_bump_version_beta()
-    new_version = android_get_app_version()
-    trigger_release_build(branch_to_build: "release/#{new_version}")
+    download_translations(app: app)
```

Comment:
I see that we use both `download_translations(options)` & `download_translations(app: app)`, I am guessing one of them is either incorrect or inconsistent. @AliSoftware Could you please take a look at this one as well?

Reply:
Both are valid and work, since in lanes the parameters you pass are in practice passed as à `Hash`. Which means that passing `(app: app)` in practice passes a dictionary with the key `app` and the appropriate value (extracted from thé `options` dictionary a couple of lines above) while passing `(options)` pass the whole dictionary of parameters the original lane was called with, which include `app` key… and maybe others.

In terms of conventions I don't mind either — passing `app: app` is explicit to only pass the needed Paramus, but passing the whole `options` is more flexible if the download_translations lane needs to extract more context one day (though… YAGNI) — but I agree we should be consistent everywhere for all calls at least.